aat/src/aat/java/uk/gov/hmcts/ccd/definitionstore/tests/functional/elasticsearch/ElasticsearchImportDefinitionTest.java

```diff
@@ -0,0 +1,200 @@
+package uk.gov.hmcts.ccd.definitionstore.tests.functional.elasticsearch;
+
+import java.io.File;
+import java.util.stream.Stream;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.Matchers.hasKey;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.http.ContentType;
+import io.restassured.response.ValidatableResponse;
+import io.restassured.specification.RequestSpecification;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.ccd.definitionstore.tests.AATHelper;
+import uk.gov.hmcts.ccd.definitionstore.tests.BaseTest;
+
+class ElasticsearchImportDefinitionTest extends BaseTest {
```

Comment:
what happens when `elasticsearch.enabled` = false?  will the tests run? we should make sure they won't, because in that case it's possible that there won't even be a cluster to run